Framework/Geometry/src/Math/Quadrilateral.cpp

```diff
@@ -11,10 +13,11 @@ using Kernel::V2D;
 //----------------------------------------------------------------------------------------------
 /** Constructor
  */
-Quadrilateral::Quadrilateral(const V2D &lowerLeft, const V2D &lowerRight,
-                             const V2D &upperRight, const V2D &upperLeft)
-    : ConvexPolygon(), m_lowerLeft(lowerLeft), m_lowerRight(lowerRight),
-      m_upperRight(upperRight), m_upperLeft(upperLeft) {}
+Quadrilateral::Quadrilateral(V2D lowerLeft, V2D lowerRight, V2D upperRight,
+                             V2D upperLeft)
+    : ConvexPolygon(), m_lowerLeft(std::move(lowerLeft)),
+      m_lowerRight(std::move(lowerRight)), m_upperRight(std::move(upperRight)),
+      m_upperLeft(std::move(upperLeft)) {}
```

Comment:
This is an example for what I mentioned in my previous comment: I suspect the move isn't any better than the copy. Actually I would not be surprised if is was slower.

(Side remark: @MichaelWedel has been looking at the performance of `Matrix` and `V3D`, and compared it to `Eigen`. I think his results show that our implementations are slow, but constructors like this are probably not the main culprit.)